messages/report.json

```diff
@@ -14,7 +14,10 @@
   "flags": {
     "jobid": "job ID of the deployment you want to check; defaults to your most recent CLI deployment if not specified",
     "wait": "wait time for command to finish in minutes",
-    "verbose": "verbose output of deploy result"
+    "verbose": "verbose output of deploy result",
+    "junit": "output JUnit test results",
+    "coverageFormatters": "output coverage results in the specified formatters",
+    "outputDir": "output directory for results"
```

Comment:
```suggestion
    "outputDir": "output directory for code coverage and JUnit results"
```

Reply:
Does this flag also default to the deploy ID?  If it does, add "; defaults to the deploy ID" to the end.